ci/Rakefile

```diff
@@ -87,6 +87,8 @@ task :pipeline, [:name] => 'pipeline.yml' do |t, args|
 
   # Default to master if the branch name is indeterminate (because HEAD is
   # detached).
+  # Set the trigger for delivery job based on which branch it is.
```

Comment:
This comment should probably be moved to immediately before line 101.

Reply:
Moved